### DISCUSSION TITLE
⚠️ 📣  YARP and YCM devel branches are no longer updated

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#372] [Announcements]
Following [this announcement](https://github.com/robotology/QA/issues/365), the `devel` branches for YARP and YCM are no longer updated. The development of new feature will happen on the `master` branch.

The `devel` branches are currently replicas of the `master` branches, but at some point, in the next few months, they will no longer be updated.

If you are using the stable branches (`yarp-3.2` and `ycm-0.10`) you should do nothing, unless you wish to update to the latest release, in which case you should switch to `yarp-3.3` and `ycm-0.11`.

If you are still on the `master` branch, you should switch *immediately* to the `yarp-3.3` and `ycm-0.11` branches.

If you are on the `devel` branch, you should switch to the `master` branch as soon as possible.

New commits on the `devel` branches of YARP and YCM will be lost, therefore please pay attention.

Pull request will be accepted for the `master` branch or for one of the stable branches.

**Comment by drdanz** [comment]
As announced long time ago, the `devel` branches for the YCM repository was removed, and the one in the YARP devel will be removed as well in the next few days.
If you still haven't done it for some reason, please switch to the correct branch as suggested in the announcement above.

**Reply by drdanz** [reply]
The stable branches are, at the moment `yarp-3.4` and `ycm-0.12`.

**Reply by pattacini** [reply]
cc @robotology/everyone

**Comment by drdanz** [comment]
YARP `devel` branch was removed as well.
